src/Tile/RadioTile.svelte

```diff
@@ -20,24 +23,31 @@
   /** Specify a name attribute for the input */
   export let name = "";
 
+  /** Obtain a reference to the input HTML element */
+  export let ref = null;
+
   import { getContext } from "svelte";
   import CheckmarkFilled16 from "carbon-icons-svelte/lib/CheckmarkFilled16";
 
-  const { add, update, selectedValue } = getContext("TileGroup");
+  const { _light, add, update, selectedValue } = getContext("RadioTileGroup");
+
+  light = light || _light;
 
   add({ value, checked });
 
   $: checked = value === $selectedValue;
 </script>
 
 <input
+  bind:this="{ref}"
   type="radio"
   id="{id}"
   name="{name}"
   value="{value}"
   checked="{checked}"
-  tabindex="{tabindex}"
+  tabindex="-1"
```

Comment:
What's the reason for changing the tabindex?

Reply:
I noticed in SelectableTile you place a -1 on the input and tabindex on the label and I thought I would align this.

Reply:
They're two different components.

The canonical implementation has it as "-1": https://github.com/carbon-design-system/carbon/blob/master/packages/components/src/components/tile/tile--selectable.hbs#L8

Reply:
OK

Reply:
Is there a rationale for one to have and the other not?